org.newerth.savagexr.json

```diff
@@ -0,0 +1,121 @@
+{
+	"app-id": "org.newerth.savagexr",
+	"branch": "master",
+	"runtime": "org.freedesktop.Platform", 
+	"sdk": "org.freedesktop.Sdk", 
+	"runtime-version": "1.6", 
+	"tags": ["proprietary"],
+	"command": "bootstrapsavagexr.sh",
+	"build-options": {
+		"cflags": "-O3 -pipe",
+		"cppflags": "-O3 -pipe"
+	},
+	"finish-args": [
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=network",
+		"--share=ipc",
+		"--device=dri",
+		"--socket=pulseaudio"
+	],
+	"modules": [
+		{
+			"name": "wget",
+			"buildsystem": "autotools",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://ftpmirror.gnu.org/wget/wget-1.19.tar.gz",
+					"sha256": "dd9523039845f69e8e945e9f2d5a38af6b1e29efa6ff53830507310235ddade8"
+				}
+			]
+		},
+		{
+			"name": "harfbuzz",
+			"config-opts": ["--disable-static",
+							"--with-gobject",
+							"--with-graphite2"
+							],
+			"sources": [
+				 {
+					 "type": "archive",
+					 "url": "http://www.freedesktop.org/software/harfbuzz/release/harfbuzz-1.7.6.tar.bz2",
+					 "sha256": "da7bed39134826cd51e57c29f1dfbe342ccedb4f4773b1c951ff05ff383e2e9b"
+				 }
+			]
+		},
+		{
+			"name": "libGLU",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "ftp://ftp.freedesktop.org/pub/mesa/glu/glu-9.0.0.tar.bz2",
+					"sha256": "1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12"
+				}
+			]
+		},
+		{
+			"name": "sdl1.2",
+			"buildsystem": "autotools",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://www.libsdl.org/release/SDL-1.2.15.tar.gz",
+					"sha256": "d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00"
+				}
+			]
+		},
+		{
+			"name": "nano",
```

Comment:
Why do you need text editor in your app?

Reply:
I've removed nano, but it would have been useful for editing some of the Savage XR config files.